File: core/src/mygdx/testone/MyGdxTestOne.java
package mygdx.testone;

import assets.images.ImagesResources;
import com.badlogic.gdx.ApplicationAdapter;
import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.graphics.GL20;
import com.badlogic.gdx.graphics.Texture;
import com.badlogic.gdx.graphics.g2d.SpriteBatch;

public class MyGdxTestOne extends ApplicationAdapter {
	SpriteBatch batch;
	Texture img,img2;
	ImagesResources imagesRes = new ImagesResources();
	
	@Override
	public void create () {
		batch = new SpriteBatch();

		img = new Texture(imagesRes.badlogicJPG());
		img2 = new Texture(imagesRes.space.ships.spaceShuttle_1PNG());
	}

	@Override
	public void render () {
		Gdx.gl.glClearColor(1, 0, 0, 1);
		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
		batch.begin();
		batch.draw(img, 0, 0);
		batch.draw(img2,100,100);
		batch.end();
	}
	
	@Override
	public void dispose () {
		batch.dispose();
		img.dispose();
	}
}

File: core/src/mygdx/testone/MyGdxTestOne.kt
package mygdx.testone

import assets.images.ImagesResources
import com.badlogic.gdx.ApplicationAdapter
import com.badlogic.gdx.Gdx
import com.badlogic.gdx.graphics.GL20
import com.badlogic.gdx.graphics.Texture
import com.badlogic.gdx.graphics.g2d.SpriteBatch

class MyGdxTestOne : ApplicationAdapter() {
    internal lateinit var batch: SpriteBatch
    internal lateinit var img: Texture
    internal lateinit var img2: Texture
    internal var imagesRes = ImagesResources()

    override fun create() {
        batch = SpriteBatch()

        img = Texture(imagesRes.badlogicJPG())
        img2 = Texture(imagesRes.space.ships.spaceShuttle_1PNG())
    }

    override fun render() {
        Gdx.gl.glClearColor(1f, 0f, 0f, 1f)
        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT)
        batch.begin()
        batch.draw(img, 0f, 0f)
        batch.draw(img2, 100f, 100f)
        batch.end()
    }

    override fun dispose() {
        batch.dispose()
        img.dispose()
    }
}
